pants.remote.ini

```diff
@@ -23,15 +23,19 @@ remote_execution_extra_platform_properties: [
     "container-image=docker://gcr.io/pants-remoting-beta/rbe-remote-execution@sha256:20046a8b909cbbd94f3ff33808ceeeaa489a05eea5d00ad57c378f81f21462fc",
   ]
 
+# This should correspond to the number of workers running in Google RBE.
+process_execution_remote_parallelism: 8
 
 [python-setup]
 interpreter_search_paths: [
+     # We include the host PATH and PEXRC values so that speculation still works.
```

Comment:
This whole block probably deserves a TODO pointing to https://github.com/pantsbuild/pants/issues/7735